package.json

```diff
@@ -26,6 +26,8 @@
     "@tanstack/react-table": "^8.9.3",
     "@wagmi/cli": "^1.3.0",
     "buffer": "^6.0.3",
+    "graphql": "^16.8.0",
```

Comment:
is this a req for `graphql-request`?

Reply:
ah yea looks like it: https://www.npmjs.com/package/graphql-request#why-do-i-have-to-install-graphql